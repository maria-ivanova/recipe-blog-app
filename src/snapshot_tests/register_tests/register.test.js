import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import Register from '../../components/register.js';

describe('Register Component', () => {
    it('should render page layout', () => {
        const tree = renderer.create(
            <Routing>
                <Register />
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})