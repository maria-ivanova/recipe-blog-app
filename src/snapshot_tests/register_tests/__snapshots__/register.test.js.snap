// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Component should render page layout 1`] = `
<section
  className="sec content_sec"
>
  <div
    className="container tcenter"
  >
    <h2
      className="page_title"
    >
      <span>
        Регистрация
      </span>
    </h2>
    <form
      className="user_form"
      onSubmit={[Function]}
    >
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Потребителско име 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="username"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Имейл 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="email"
          onChange={[Function]}
          required={true}
          type="email"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Парола 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="password"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <label
          className="lbl"
        >
          Повтори паролата 
          <span
            className="tred"
          >
            *
          </span>
        </label>
        <input
          className="input_text input_text"
          name="rePassword"
          onChange={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="input_row"
      >
        <button
          className="btn"
          type="submit"
        >
          Регистрирай се
        </button>
      </div>
    </form>
  </div>
</section>
`;
